src/frontend/components/Banner/Banner.tsx

```diff
@@ -3,16 +3,23 @@
 
 import Link from 'next/link';
 import * as S from './Banner.styled';
+import Bugsnag from '@bugsnag/js';
 
 const Banner = () => {
+  const onButtonClick = () => {
+    Bugsnag.notify(new Error('Test error'));
```

Comment:
Sending an error each time we click this button might not be optimal. 
@mottibec Did you have anything specific scenario in mind for errors in the frontend?
I wonder if we could add an additional flag for it and only fire it then. Or for now, add a dedicated button somewhere on the page to generate errors on click?